src/shared/PSAModal.js

```diff
@@ -0,0 +1,24 @@
+/*
+  This file contains the Modal Component
+*/
+
+import React from 'react';
+import { Modal } from '@mui/material';
+
+const PSAModal = ({
+  data, sx, style, open, onClose, closeAfterTransition, disableEscapeKeyDown,
+}) => (
+  <Modal
+    sx={sx}
+    open={open}
+    style={style}
+    onClose={onClose}
+    closeAfterTrantion={closeAfterTransition}
+    disableEscapeKeyDown={disableEscapeKeyDown}
+  >
+    {data}
```

Comment:
Can we change the name of this property from data to modalContent.  I think it will be a little more clear what it is and differentiate itself from "data" that we get as a response from an  API call.